src/main/java/com/olgaboiar/mint/Router.java

```diff
@@ -3,43 +3,57 @@
 import com.olgaboiar.mint.handlers.*;
 
 import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
 
 public class Router {
     private Map<String, Map<String, IHandler>> routes = new HashMap<>();
+//    Map<String, IHandler> methodHandlers;
 
     public Router() {
         registerTestRoutes();
     }
 
     public Response route(Request request) throws IOException {
-        Map<String, IHandler> methodList = routes.get(request.getRequestedFile());
-        if (methodList == null) {
+        Map<String, IHandler> methodHandlers = routes.get(request.getUri());
+        if (methodHandlers == null) {
```

Comment:
Is there a built-in method that checks if a map is empty? For example methodHandlers.isEmpty() I'm not sure of the correct syntax for Java.